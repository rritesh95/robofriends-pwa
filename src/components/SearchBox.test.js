import React from 'react';
import { shallow } from 'enzyme';
import SearchBox from './SearchBox';

it('expect to render searchBox component', () => {
    const mockSearchChange = () => {
        console.log("search box changed");
    };

    expect(shallow(<SearchBox searchChange={mockSearchChange}/>)).toMatchSnapshot();
})