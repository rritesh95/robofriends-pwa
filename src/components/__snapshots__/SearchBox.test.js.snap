// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render searchBox component 1`] = `
<div
  className="pa2"
>
  <input
    aria-label="Search"
    className="pa3 ba b--green bg-lightest-blue"
    onChange={[Function]}
    placeholder="search robots"
    type="search"
  />
</div>
`;
